searx/settings.yml

```diff
@@ -648,6 +648,64 @@ engines:
     shortcut : qws
     categories : social media
 
+    # recoll is a local search engine based on Xapian:
```

Comment:
Would it better to add this documentation into docs/admin/engines/recoll.rst ?
So settings.yml stay "light".

Reply:
Where should I put it? AFAIK we don't have special documentation for engines.

Reply:
I vote for adding a subsection to [admin/engines](https://github.com/searx/searx/blob/master/docs/admin/engines.rst)  for each engine which needs more documentation. This can also be done by inserting a `.. toctree::` at the top and adding a file for each engine.  If [reST](https://searx.github.io/searx/dev/reST.html) is not your part, I can prepare a patch on top of this PR.

Reply:
@return42 I would be grateful if you would prepare a PR with the proper page layout. Structuring documentation is not my strong suit. I have added a new page with the description of the engine, so you do not have to write that yourself.

Reply:
I prepared to patches (e9134fa77c6 and 7e69765) you can pull from my [recollengine](https://github.com/return42/searx/tree/recollengine) branch.

> Structuring documentation is not my strong suit.

No problem, we all have our own majors and minors interests .. if you like to see other things documented, ask me.

Reply:
I have picked the patches. Thank you!